content/docs/start/visualization.md

```diff
@@ -0,0 +1,109 @@
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from text with DVC
+
+A useful plot to show the classification performance is the confusion matrix. In
+order to produce it, DVC expects a CSV **plot file** in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+> We added a [loop] comparing the results to generate this file from the
+> predictions.
+
+[loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L123
+
+After running the experiment with `dvc exp run`, DVC will produce
+`plots/confusion.csv` file presenting the classifier performance and open it in
+the browser:
+
+```dvc
+$ dvc plots show plots/confusion.csv --template confusion \
+                                     -x actual -y predicted
+file:///Users/.../example-dvc-experiments/plots/confusion.json.html
+```
+
+![confusion matrix](/img/start_visualization_confusion1.png)
+
+### Using plots generated by scripts
+
+Let's produce another plot to see misclassified examples from each class. This
+procedure generates the misclassification examples from the validation data and
+arranges them into a _confusion table_ that shows the correct label, and
+misclassification sample. The code to generate an image from a set of training
+images is omitted here but you can find the code in [the example
+project.][misclassified-example-code]
+
+[misclassified-example-code]:
+  https://github.com/iterative/example-dvc-experiments/blob/48b1e5078c957f71674c00f416290eaa3b20b559/src/util.py#L49
+
+DVC automatically shows the image in an HTML page with the command:
+
+```dvc
+$ dvc plots show plots/confusion.png
+```
+
+![confusion image](/img/start_visualization_confusion2.png)
+
+### Autogenerating plots with integrations
+
+An important issue for deep learning projects is to observe in which epoch
+training loss and validation loss differs. DVC helps in that regard with its
+integrations to deep learning libraries via DVCLive.
+
+The example project uses Keras to train a classifier, and we have a DVCLive
+callback that visualizes the training and validation loss for each epoch. We
+first import the callback from DVCLive.
+
+```python
+from dvclive.keras import DvcLiveCallback
+```
+
+Then we add this callback to `fit` callbacks.
+
+```python
+model.fit(
+ ...
+ callbacks=[DvcLiveCallback()],
+ ...)
+```
+
+With these two changes, we can see the plots showing the defined metrics in our
+model. You can see the HTML report with your browser in
+`training_metrics/index.html` file.
+
+![dvclive](/img/start_visualization_dvclive.png)
+
+DVCLive has other capabilities, like saving the model every epoch or modifying
+these default values. Please refer to [DVClive documentation] for details.
+
+[dvclive documentation]: /doc/dvclive/dvclive-with-dvc
```

Comment:
We'd already have a dvclive link early in the section with one of my suggestions.
```suggestion
these default values.
```